app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoiceVirtualProductCustomStock.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCreateCreditMemoTotalRefundFullInvoiceVirtualProductCustomStock" extends="AdminCreateCreditMemoTotalRefundFullInvoiceVirtualProductDefaultStock">
+        <annotations>
+            <stories value="Virtual Product Custom Stock."/>
+            <title value="Create Total Refund Credit Memo after Full Invoice With Virtual Product on Custom Stock."/>
+            <description value="Verify admin able to create Credit Memo with total refund after full invoice with virtual product custom stock."/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735141/scenarios/1695035"/>
```

Comment:
* AdminCreateCreditMemoTotalRefundVirtualProductDefaultStock
* AdminCreateCreditMemoTotalRefundVirtualProductCustomStock

Reply:
Same as above, not able to find AdminCreateCreditMemoTotalRefundVirtualProductDefaultStock and AdminCreateCreditMemoTotalRefundVirtualProductCustomStock on 2.3-develop